plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -31,21 +49,77 @@ impl FromStr for CgroupV2Metric {
             time_used_tot: 0,
             time_used_user_mode: 0,
             time_used_system_mode: 0,
+            anon_used_mem: 0,
+            file_mem: 0,
+            kernel_mem: 0,
+            pagetables_mem: 0,
         };
+
         for line in s.lines() {
             let parts: Vec<&str> = line.split_ascii_whitespace().collect();
             if parts.len() >= 2 {
-                match parts[0] {
-                    "usage_usec" => {
-                        cgroup_struc_to_ret.time_used_tot = parts[1].parse::<u64>()?;
-                    }
-                    "user_usec" => {
-                        cgroup_struc_to_ret.time_used_user_mode = parts[1].parse::<u64>()?;
+                // Parse value as f64
+                match parts[1].parse::<f64>() {
+                    Ok(value) => {
+                        // Check if value is unsigned
+                        let res: u64 = if value < 0.0 || value.fract() != 0.0 {
+                            0
+                        } else {
+                            value as u64
+                        };
```

Comment:
Is there a reason for parsing as `f64` and then only keeping values that are unsigned integers?